### PR TITLE
Adds realtime validations to register form.

### PR DESCRIPTION
**Pull request by adeelkhan**
This patch adds realtime/ validations for;
	*) name
	*) fullname
	*) password
	*) email

VAN-30

### CODE REVIEW

**Review by waheedahmed — COMMENTED** [review]
With these real-time validations from the backend, I think we can remove the front validations because these are conflicting with the backend error messages and also doesn't match the monolith experience.

File: src/logistration/RegistrationPage.jsx [hunk 1]
```diff
@@ -36,6 +41,9 @@ class RegistrationPage extends React.Component {
   constructor(props, context) {
     super(props, context);
 
+    const { intl } = props;
+    this.intl = intl;
```
[the remark this answers is not in the document]
Comment (adeelkhan) [reply src/logistration/RegistrationPage.jsx]: As far as the front end validations are concerned. We can discuss this and create a separate ticket to remove this code.

File: src/logistration/tests/RegistrationPage.test.jsx [hunk 2]
```diff
@@ -237,6 +237,62 @@ describe('./RegistrationPage.js', () => {
...
+    expect(store.dispatch).toHaveBeenCalledWith(fetchRealtimeValidations(formPayload));
+
+    registrationPage.find('input#email').simulate('blur', { target: { value: '', name: 'email' } });
+    expect(store.dispatch).toHaveBeenCalledWith(fetchRealtimeValidations(formPayload));
+
+    registrationPage.find('input#password').simulate('blur', { target: { value: '', name: 'password' } });
+    expect(store.dispatch).toHaveBeenCalledWith(fetchRealtimeValidations(formPayload));
+  });
+
+  it('should dispatch registerNewUser on Submit', () => {
```
Comment (waheedahmed) [inline src/logistration/tests/RegistrationPage.test.jsx:273]: shouldn't it be `should not dispatch registerNewUser on Submit`?

**Review by waheedahmed — APPROVED** [review]
One minor comment otherwise 👍

**Pull request merged**